### ISSUE TITLE
skein.driver shut down

### ISSUE THREAD

**Issue by GetDarren** [dask/dask-yarn#67]
Here is my submit:
dask-yarn submit --environment dask_venv.tar.gz --worker-count 8 --worker-vcores 2 --worker-memory 4GiB --scheduler-vcores 2 --scheduler-memory 4GiB test_submit.py

Here is my test code:
```
from sklearn.ensemble import GradientBoostingClassifier
import sklearn.datasets
import dask_ml.datasets
from dask_ml.wrappers import ParallelPostFit
import time
from dask_yarn import YarnCluster
from dask.distributed import Client

cluster = YarnCluster.from_current()
client=Client(cluster)

cluster.scale(10)

X,y = sklearn.datasets.make_classification(n_samples = 1000, random_state=0)

clf = ParallelPostFit(estimator=GradientBoostingClassifier())
clf.fit(X,y)

X_big, _ = dask_ml.datasets.make_classification(n_samples = 2000000, chunks=200000, random_state=0)

t1 = time.time()
clf.predict(X_big).compute()
print("Time dask:{}".format(time.time()-t1))
dask_time = time.time() - t1

clf_local = GradientBoostingClassifier()
clf_local.fit(X,y)

t2 = time.time()
clf_local.predict(X_big)
print("Time sklearn:{}".format(time.time()-t2))
sklearn_time = time.time() - t2

time = [dask_time,sklearn_time]

with open('test.txt','w') as f:
    for item in my_list:
        f.write("%s\n" % item)
```

Here is my error:

```
19/05/29 02:46:07 INFO client.RMProxy: Connecting to ResourceManager at gce-europe-west2-a-d-dataproc-sql-debug6-m/192.168.3.40:8032
19/05/29 02:46:07 INFO client.AHSProxy: Connecting to Application History server at gce-europe-west2-a-d-dataproc-sql-debug6-m/192.168.3.40:10200
19/05/29 02:46:08 INFO skein.Driver: Driver started, listening on 35151
19/05/29 02:46:08 INFO skein.Driver: Uploading application resources to hdfs://gce-europe-west2-a-d-dataproc-sql-debug6-m/user/d5054581/.skein/application_1557133139144_0344
19/05/29 02:46:09 INFO skein.Driver: Submitting application...
19/05/29 02:46:10 INFO impl.YarnClientImpl: Submitted application application_1557133139144_0344
application_1557133139144_0344
19/05/29 02:46:10 INFO skein.Driver: Driver shut down
```
Here is the dask-yarn status:
Here is the dask-yarn status:
```
19/05/29 03:04:23 INFO skein.Driver: Driver started, listening on 39537
19/05/29 03:04:24 INFO skein.Driver: Driver shut down
APPLICATION_ID                    NAME    STATE       STATUS    CONTAINERS    VCORES    MEMORY    RUNTIME
application_1557133139144_0343    dask    FINISHED    FAILED    0             0         0         17s
```

And I cannot find the application logs:
```
19/05/29 03:02:35 INFO client.RMProxy: Connecting to ResourceManager at gce-europe-west2-a-d-dataproc-sql-debug6-m/192.168.3.40:8032
19/05/29 03:02:35 INFO client.AHSProxy: Connecting to Application History server at gce-europe-west2-a-d-dataproc-sql-debug6-m/192.168.3.40:10200
File /yarn-logs/d5054581/logs/application_1557133139144_0343 does not exist.
Can not find any log file matching the pattern: [ALL] for the application: application_1557133139144_0343
Can not find the logs for the application: application_1557133139144_0343 with the appOwner: d5054581
```

Could you help to check the reason?
Many Thanks

**Comment by jcrist** [comment]
I'm sorry, without the application logs I can't help debug. I'm not sure why you're unable to retrieve them, but one issue may be the incorrect application number (it was `application_1557133139144_0344` above, but you asked for `application_1557133139144_0343` when getting the logs).

**Comment by jcrist** [comment]
Closing as stale, feel free to reopen if the issue persists.
